.schema/config.schema.json

```diff
@@ -260,6 +260,13 @@
           ],
           "default": 302
         },
+        "url_param": {
+          "title": "URL query parameter",
+          "description": "Name of query parameter, where current url will be added to redirect url",
+          "type": "string",
+          "pattern": "^[A-Za-z0-9,._~-]*$",
```

Comment:
Can you maybe give a pointer to where this pattern comes from? Isn't it just "not space"?

Reply:
Took it from here: https://www.456bereastreet.com/archive/201008/what_characters_are_allowed_unencoded_in_query_strings/#:~:text=According%20to%20section%203.4%20Query,%3F%E2%80%9D%20are%20allowed%20as%20well.
it cites RFC 3986

Reply:
:+1: